dags/overwatch.py

```diff
@@ -0,0 +1,53 @@
+"""
+Overwatch
+Runs daily at 0700 UTC
+
+Source code is [overwatch repository](https://github.com/mozilla/overwatch/).
+"""
+
+from datetime import datetime, timedelta
+
+from airflow import DAG
+from airflow.models import Variable
+from operators.gcp_container_operator import GKEPodOperator
+
+docs = """\
+This DAG is executes the Overwatch monitoring system.
+All alerts related to this DAG can be ignored.
+
+(for more info on dim see: https://github.com/mozilla/overwatch-mvp)
+"""
+
+
+default_args = {
+    "owner": "gleonard@mozilla.com",
+    "email": [
+        "gleonard@mozilla.com",
+        "wichan@mozilla.com",
+    ],
+    "start_date": datetime(2022, 11, 1),
+    "depends_on_past": False,
+    "email_on_failure": True,
+    "email_on_retry": True,
+    "retries": 0,
+}
+
+tags = ["repo/telemetry-airflow", "impact/tier_3", ]
+image = "gcr.io/moz-fx-data-airflow-prod-88e0/overwatch:" + Variable.get("overwatch_image_version")
```

Comment:
nit:

```suggestion
image = "gcr.io/moz-fx-data-airflow-prod-88e0/overwatch:{{ var.value.overwatch_image_version }}"
```

Reply:
If I recall correctly. Every time the DAG is parsed this logic would be executed. If using the macro instead the value should only be retrieved when the task is initiated.